atlas/src/shared/components/Carousel/Carousel.tsx

```diff
@@ -1,94 +1,52 @@
-import React, { useState } from 'react'
-import { SerializedStyles } from '@emotion/core'
-import { animated, useSpring } from 'react-spring'
-import useResizeObserver from 'use-resize-observer'
-import { CarouselStyleProps, useCSS } from './Carousel.style'
-import NavButton from '../NavButton'
+import React from 'react'
+import styled from '@emotion/styled'
+import Glider, { GliderProps, BreakPoint } from 'react-glider'
+import 'glider-js/glider.min.css'
 
-export type CarouselProps = {
-  containerCss?: SerializedStyles
-  leftControlCss?: SerializedStyles
-  rightControlCss?: SerializedStyles
-  disableControls?: boolean
-  onScroll?: (direction: 'left' | 'right') => void
-} & CarouselStyleProps
+import Icon from '../Icon'
 
+export type CarouselProps = {
+  trackPadding?: string
+} & GliderProps
+
+const Track: React.FC<any> = ({ className = '', ...props }) => (
+  <div className={`glider-track ${className}`} {...props} />
+)
+
+const defaultBreakpoints: BreakPoint[] = [
+  {
+    breakpoint: 400,
```

Comment:
At this point, it would probably make sense to define some app-wide breakpoints. We have those and also the breakpoint for the navbar, I'm sure we'll have some more. What do you think?